spec/features/admin_views_users_spec.rb

```diff
@@ -0,0 +1,93 @@
+require "rails_helper"
+
+feature "Admin views a list of users", %{
+  As an admin,
+  I want to view a list of users
+  So I know who is on my site
+  } do
+    
+  # Acceptance Criteria:
+  # [X] I must be logged in, and an admin to view the list of users
+  # [X] Every user is listed alphabetically by username
+  # [X] When I click on a username, I am taken to their user profile
+  # [X] I can get to this page by clicking on the option in the admin homepage
+
+  context "Admin is signed in" do
+    before(:each) do
+      @admin1 = FactoryGirl.create(:user, role: "admin")
+
+      visit root_path
+
+      click_on "Sign In"
+
+      fill_in "Login", with: @admin1.email
+      fill_in "Password", with: @admin1.password
+
+      click_on "Log in"
+    end
+
+    scenario "Admin follows paths to view a user's profile" do
+      user1 = FactoryGirl.create(:user, username: "Cori")
```

Comment:
explicitly setting the username could result in uniqueness constraints getting triggered. Just utilize the username that is assigned in the factory.

```ruby
 expect(find("ul#users li:nth-child(1)")).to have_content user1.username
```